src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,43 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static alive(animals) {
```

Comment:
Maybe you mean `getAlive`, that would be more descriptive.
I don't see where you are using this method. If you don't use it, then delete it to avoid confusion.